test/spec/PasswordReset_spec.js

```diff
@@ -79,6 +79,12 @@ function (Q, _, $, OktaAuth, LoginUtil, Util, PasswordResetForm, Beacon, Expect,
           .toEqual(['', { trigger: true }]);
       });
     });
+    itp('has a valid subtitle', function () {
```

Comment:
I'd expect there to be several more tests - if you add logic, test it. For example:
- What happens when they only have one?
- What happens if there are none?